library/Fission/Platform/User.hs

```diff
@@ -15,7 +15,7 @@ import Data.Swagger as Swagger
 import Fission.Internal.JSON
 import Fission.Security
 
-data UserConfig = UserConfig
+data Config = Config
```

Comment:
🤔 Perhaps `Provision` as in "provision response". You're right that we can use this to provision via the CLI (for example), not just Heroku. `Config` tends to  mean the `cfg` in `MonadRIO cfg m`. Thoughts?